### ISSUE TITLE
Run user script on detection

### ISSUE THREAD

**Issue by ArkadiuszRaj** [fripon/freeture#21]
When event is detected it is convinient to run some script, like automatic uploader.
Here is example I use in my local build of freeture.

>  // ****************************\*  starting user script  *****************************
>         if(boost::filesystem::exists(mdtp.DET_EVENT_USER_ACTION) ){
>             string commandString = mdtp.DET_EVENT_USER_ACTION + " " + mEventPath;
>             BOOST_LOG_SEV(logger,notification) << "Executing user command : " << commandString;
>             std::thread ut([this,commandString](){ system(commandString.c_str()); });
>             ut.detach();
>         }

**Comment by cmarmo** [comment]
Hi,
this was a Christmas wish... :)
Thanks for your code, I will try to test it ASAP... but please, be patient :)

**Comment by ArkadiuszRaj** [comment]
See https://github.com/fripon/freeture/pull/24

**Comment by ArkadiuszRaj** [comment]
Hi @MatthieuGarnung can you push for merging this PR #24 ??
